Update access_pattern_chkpt.md

nphtan opened the pull request:
2024 JLESC Access Pattern Checkpointing project update.

nphtan committed:
Update access_pattern_chkpt.md

JLESC 16 (2024) Access Pattern Checkpointing project update.

pancetta commented:
Please add citations to the project bibfile (if this was not done within the JLESC) or the [JLESC bib file](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) (if this should be counted as JLESC publication) and cite from there.

nphtan committed:
Update jlesc.bib

Add Scalable Incremental Checkpointing using GPU-Accelerated De-Duplication bibtex entry.

nphtan committed:
Update access_pattern_chkpt.bib

Add bibtex entry to project specific bib file.

nphtan committed:
Update access_pattern_chkpt.bib

Remove unnecessary external entry from access_pattern_chkpt.bib

nphtan committed:
Update access_pattern_chkpt.md

Adjust access_pattern_chkpt.md to use jlesc.bib for citations.

nphtan committed:
Fix bibtex entry for TanEtAl2023.

nphtan committed:
Fix spacing for call to generate references.

nphtan commented:
I have added the citations for the publications and adjusted the project file accordingly.

pancetta committed:
Merge branch 'source' into patch-2

nphtan committed:
Fix bibtex entry merge issue.

Fix merge issue that removed a closing curly brace.

nphtan commented:
This should fix the merge problem.
